### DISCUSSION TITLE
Display details by defaut

### DISCUSSION THREAD

**Discussion by Warkorentin** [protz/thunderbird-conversations#1817] [Q&A] [answered]
Hey, 

Do you guys know if there is a way to have the "show details" button on by default ? The one in the top-right corner of each conversation. 

I don't really want to click it every time I receive an email. But I don't see any other good option to knowing whether I'm the direct recipient or simply in cc. 

Tanks !

**Comment by Standard8** [comment, marked as answer]
Go into the Thunderbird's preferences, and at the bottom of the "General" section there's a Config Editor button. Go into that and acknowledge the warning then search for `mail.show_headers`. Double click the value (probably the default of 1) and set that to 2.

**Reply by Warkorentin** [reply]
Awesome ! Thanks a lot !
